make-distribution.sh

```diff
@@ -121,7 +121,7 @@ if [ $(command -v git) ]; then
 fi
 
 
-if [ ! $(command -v "$MVN") ] ; then
+if [ ! "$(command -v "$MVN")" ] ; then
```

Comment:
Yeah I tried it locally too. Just making sure the first two double qutoes didn't somehow bind more closely than the parens.